lib/net/modules/dns.toit

```diff
@@ -674,58 +676,58 @@ create-dns-packet queries/List records/List -> ByteArray
 
   previous-locations := {:}
 
-  result := Buffer
+  result := io.Buffer
 
   status-bits := is-response ? 0x8000 : 0x0000
   if is-authoritative: status-bits |= 0x0400
 
-  result.write-int16-big-endian id
-  result.write-int16-big-endian status-bits
-  result.write-int16-big-endian queries.size
-  result.write-int16-big-endian records.size
-  result.write-int32-big-endian 0  // Don't support the other things.
+  result.big-endian.write-int16 id
```

Comment:
Is it possible to cache result.big-endian in a variable to avoid it being looked up on every single line?

Reply:
done.